backend/composer/api/serializers.py

```diff
@@ -486,25 +486,39 @@ def get_statement_preview(self, instance):
             self.context['journey'] = instance.get_journey()
         return self.create_statement_preview(instance, self.context['journey'])
 
+    def get_property_or_empty_string(self, instance, property_name, sub_property_name=None):
+        if hasattr(instance, property_name):
+            property_attr = getattr(instance, property_name)
+            if property_attr:
+                if sub_property_name and hasattr(property_attr, sub_property_name):
+                    subproperty_attr = getattr(property_attr, sub_property_name)
+                    return subproperty_attr() if callable(subproperty_attr) else subproperty_attr
+                
+                if callable(property_attr):
+                    return property_attr() if property_attr() else ""
```

Comment:
@D-GopalKrishna why do we return "" instead of None later on on line 527 you can test against `is None`